platforms/hyperledger-besu/configuration/reset-network.yaml

```diff
@@ -26,6 +26,22 @@
         kubernetes: "{{ item.k8s }}"
       loop: "{{ network['organizations'] }}"
 
+    ## Delete ambassador tls certificate for node
+    - name: "delete ambassador tls certificates for Nodes"
```

Comment:
Please rename it to `delete ambassador tls certificates for organizations`

Reply:
renamed, licence patch added to all files, cert-manager chart has been packaged